components/ProjectDetails.js

```diff
@@ -1,10 +1,16 @@
 import styled from "styled-components";
 import Image from "next/image";
 import ModalDelete from "@/components/ModalDelete";
+import ModalEdit from "@/components/ModalEdit";
 import { useState } from "react";
 import router from "next/router";
 
-export default function ProjectDetails({ currentProject, onDeleteProject }) {
+export default function ProjectDetails({
+  currentProject,
+  onDeleteProject,
+  onEditProject,
+}) {
+  const [modalEdit, setModalEdit] = useState(false);
```

Comment:
for boolean choose is...

Reply:
Good hint! We will fix/refactor this at a later point, since we need to change this prefixing in a couple of components/files.